### DISCUSSION TITLE
Create A Postman-like Frontend For Testing and Debugging tRPC APIs

### DISCUSSION THREAD

**Discussion by clostao** [trpc/trpc#5675] [Ideas]
One of the advantages of tRPC is the strong-typed capabilities and known schema of the API. I propose to build a tRPC project new frontend. 

This frontend could take the API"s kwown schema and generate a dynamic set of forms for interacting with the API. This would ease testing and debugging this kind of APIs.

**Comment by david8z** [comment]
Have you tried [trpc-panel](https://github.com/iway1/trpc-panel)?

**Reply by olitreadwell** [reply]
trpc-panel seems like an order of magnitude more complex for someone who is just getting started with tRPC.
